src/cell_group.hpp

```diff
@@ -65,23 +129,24 @@ class cell_group {
         spikes_.clear();
         clear_events();
         reset_samplers();
-        cell_.reset();
+        binner_.reset();
+        lowered_.reset();
     }
 
-    time_type min_step(time_type dt) {
-        return 0.1*dt;
+    void set_binning_policy(binning_kind policy, time_type bin_interval) {
+        binner_ = event_binner(policy, bin_interval);
```

Comment:
`binner_` not used yet... next (rather intrusive) PR will use the binner and expose the policy.